esp/src/package.json

```diff
@@ -27,8 +27,7 @@
     "dev-start-ws": "ws",
     "dev-start": "run-p bundle-watch dev-start-ws",
     "dev-start-verbose": "ws --verbose.include request response",
-    "dev-link": "rimraf ./node_modules/@hpcc-js && mklink /J .\\node_modules\\@hpcc-js ..\\..\\..\\hpcc-js\\packages",
-    "dev-unlink": "rimraf ./node_modules/@hpcc-js && npm install",
+    "rm-hpcc": "rimraf ./node_modules/@hpcc-js",
```

Comment:
Didn't @xwang2713 have a problem with this npm command when running docker with removing a dependency at one point? Will this have a negative effect as well?

Reply:
I don't think so:
1. You have to manually call `npm run rm-hpcc` it doesn't get called anywhere automatically
2. To restore the machine back to "normal" you just call:  `npm install`

I don't really expect anyone to be availing of this except myself.